kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jSinkTask.kt

```diff
@@ -7,35 +7,48 @@ import org.apache.kafka.connect.sink.SinkRecord
 import org.apache.kafka.connect.sink.SinkTask
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
+    import streams.service.errors.*
 import streams.utils.StreamsUtils
+import java.lang.Exception
+import java.util.*
 
 
 class Neo4jSinkTask : SinkTask() {
     private val log: Logger = LoggerFactory.getLogger(Neo4jSinkTask::class.java)
     private lateinit var config: Neo4jSinkConnectorConfig
     private lateinit var neo4jService: Neo4jService
+    private lateinit var errorService: ErrorService
 
     override fun version(): String {
         return VersionUtil.version(this.javaClass as Class<*>)
     }
 
     override fun start(map: Map<String, String>) {
         this.config = Neo4jSinkConnectorConfig(map)
+
+        val kafkaConfig = Properties()
+        kafkaConfig.putAll(map)
+        this.errorService = KafkaDLQService(kafkaConfig, ErrorService.ErrorConfig.from(kafkaConfig), log::error)
```

Comment:
This is another instance of inconsistent third argument (log::error) to KafkaDLQService.  See earlier comment.

Reply:
see my earlier answer :)